### ISSUE TITLE
Local abstraction not identical with remote abtraction

### ISSUE THREAD

**Issue by curiosityseeker** [roddhjav/apparmor.d#344]
I'm surprised to see that https://github.com/roddhjav/apparmor.d/blob/main/apparmor.d/abstractions/app/firefox contains the following code:

```
@{lib_dirs}/{,**}             r,
@{lib_dirs}/*.so              mr,
@{lib_dirs}/crashreporter     rPx,
@{lib_dirs}/glxtest           rPx -> firefox//&firefox-glxtest,
@{lib_dirs}/minidump-analyzer rPx,
@{lib_dirs}/pingsender        rPx,
@{lib_dirs}/plugin-container  rPx,
@{lib_dirs}/vaapitest         rPx -> firefox//&firefox-vaapitest,
```

However, the same code part on my system looks like this:
```
@{lib_dirs}/{,**}             r,
@{lib_dirs}/*.so              mr,
@{lib_dirs}/crashreporter     rpx,
@{lib_dirs}/glxtest           rpx -> firefox//&firefox-glxtest,
@{lib_dirs}/minidump-analyzer rpx,
@{lib_dirs}/pingsender        rpx,
@{lib_dirs}/plugin-container  rpx,
@{lib_dirs}/vaapitest         rpx -> firefox//&firefox-vaapitest,
```

So all `rPx,` rules are changed to `rpx,` rules. How come? ( I haven't checked so far if the same observation can be made for other profiles/abstractions.)

**Comment by roddhjav** [comment]
This is currently done on purpose as it the actual working fix for #80 and #235. The problem is that some programs do not start well while scrubing the environment (including dbus but not limited to it). The temporary solution is to on build time, convert all `Px` to `px` while we find out a proper solution.

 See 334c930969db89f3f85b002da5c2f69e5277973b

**Comment by curiosityseeker** [comment]
Ah, I see! I missed that due to my long absence. Thanks for the clarification!
